packages/plus/src/components/Navigation/NavigationContent.tsx

```diff
@@ -316,6 +316,9 @@ export const NavigationContent = ({
                   sentiment="neutral"
                   size="small"
                   icon={expanded ? 'arrow-left-double' : 'arrow-right-double'}
+                  aria-label={
+                    expanded ? 'Collapse navigation' : 'Expand navigation'
```

Comment:
no variable to pass translation ? agree its probably over loaded, just wondering what is the practice so far?

Reply:
true I will reuse the one from tooltip